StyleGuide.md

```diff
@@ -542,6 +543,92 @@ There is also a custom purple callout that uses the code `{{site.data.alerts.cal
 
 Each Liquid tag should be on its own line. You can use Markdown within the highlighted text.
 
+### Known limitations
+
+#### What are known limitations?
+
+Sometimes CockroachDB does not behave the way that users expect it to behave. These deviations from expected behavior can be in the form of:
+
+- A difference in syntax between CockroachDB and [SQL Standard](https://blog.ansi.org/2018/10/sql-standard-iso-iec-9075-2016-ansi-x3-135)
+- A difference in the behavior of CockroachDB and PostgreSQL
+- A feature that is functional, but not yet fully implemented
+- A feature that is fully implemented, but has some long-standing bugs
+- A feature that limits performance
+
+We list the general differences between CockroachDB and the SQL Standard/PostgreSQL on our [SQL Feature Support](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html) page, and we provide more details on the differences between CockroachDB and PostgreSQL on our [PostgreSQL Compatibility](https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html). All other instances of known, but possibly unexpected, database behavior are known as *known limitations*.
+
+Known limitations often have [associated GitHub issues in the `cockroach` repo](https://github.com/cockroachdb/cockroach/issues), meaning the limitation could be resolved one day (in theory...). *Not all known limitations have GitHub issues, and not all known limitations will be resolved.*
+
+The purpose of documenting a known limitations is to help our users know more about using our product safely and effectively.
+
+#### Where to find known limitations
+
+Known limitations are generally listed in two places:
+
+1. (More common) In the `cockroach` repo, as [open issues with the `docs-known-limitations` label, but *not* with the `docs-done` label](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+label%3Adocs-known-limitation+-label%3Adocs-done+is%3Aopen). Usually, engineers add these labels to issues in the weeks leading up to the release.
+
+1. (Less common) In the `docs` repo, as [open issues with the `T-known-limitation` label](https://github.com/cockroachdb/docs/issues?q=is%3Aopen+is%3Aissue+label%3AT-known-limitation).
+
+If you come across some behavior that you believe qualifies as a known limitation, first open an issue in the `cockroach` repo, get some engineering/PM feedback on the issue, and then add a `docs-known-limitations` label to an issue.
+
+#### When to document known limitations
+
+Documenting known limitations should happen in the [weeks leading up to a GA release](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/402718726/GA+Release+Checklist).
+
+*You can document a known limitation too early (usually in the event that the limitation is a bug that engineering finds the time to fix during the stability period leading up to a GA release).*
```

Comment:
I would edit this to be a little more directive, to something like "avoid documenting a known limitation too early, if possible ..."

(I know what you mean tho)

Reply:
Done.